.github/dependabot.yml

```diff
@@ -16,25 +16,17 @@ updates:
           - ">= 0.8.0"
           - "< 99"
 
-      # Jest upgrade is blocked because the vue-test-utils don't support it yet for Vue 2
-      - dependency-name: jest
-        versions:
-          - ">= 27"
-
-      # Until we are ready to migrate to Vue 3
-      - dependency-name: vue
-        versions:
-          - ">= 3"
-
-      # Is Vue 3 only
-      - dependency-name: vue-loader
-        versions:
-          - ">= 16"
-
       # Can't be used as CommonJs but only ESM which will break our FE build atm
       - dependency-name: chalk
         versions:
           - ">= 5"
+
+      # This dependencies have to be updated according to the needs of @masterportal/masterportalapi
+      - dependency-name: ol
```

Comment:
How does this ensure that they are only updated together?

Reply:
including them here without specifying the "versions" key leads to dependabot excluding them entirely, so this does only ensure they are never updated by dependabot.

Reply:
Can't we create a group with masterportalapi so dependabot creates updates for the group?

Reply:
unfortunately no. masterportalapi is not provided by npm. so we have to check for updates manually (As far as I could see bitbucket is not availalble as depenabot source - https://docs.github.com/en/code-security/dependabot/ecosystems-supported-by-dependabot/supported-ecosystems-and-repositories).
With that, we have to check for masterportal-updates manually and when we update manually, we have to adjust the dependecies too. And: with almost every masterportal update olcs comes with another issue regarding import resolution :-/ that has to be handled manually anyway

Reply:
Right :see_no_evil:  Could we add that info here (masterportalapi is not provided by npm, so we have to check for updates manually, and we need to update these dependencies along with masterportalapi)? Or is this documented somewhere else?

Reply:
like so 40e191cff4c782456c9fc5ad97ca527778df63ea